tests/spec/testcase.rs

```diff
@@ -25,7 +25,7 @@ impl<'a> SpecTestCase<'a> {
             let json = v
                 .into_iter()
                 .find(|x| x["description"].as_str().unwrap().eq(desc))
-                .unwrap();
```

Comment:
This kind of helped grok how to make changes to the test suite, so figured I'd offer it up to give any future new contributors some more context than just a generic error message.